### PR TITLE
[Feature] require-returns

### PR DESCRIPTION
**Pull request by ptkdev**
**Feature**: require-returns
**Description**: Requires returns parameters are documented.
**Test**: passed
**Documentation**: writed

**Fix bug**: #137 and #14

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,7 +57,7 @@
     "add-assertions": "babel-node ./src/bin/readme-assertions",
     "build": "rm -fr ./dist && NODE_ENV=production babel ./src --out-dir ./dist --copy-files --source-maps",
     "create-readme": "gitdown ./.README/README.md --output-file ./README.md && npm run add-assertions",
-    "lint": "eslint ./src ./test",
+    "lint": "eslint ./src ./test --fix",
```
Comment (gajus) [inline package.json]: Remove `--fix`. Does not belong in this PR.

**Comment by ptkdev** [comment]
Done! Correct 👍

**Comment by ptkdev** [comment]
@gajus done! Fixed :)

**Comment by gajus** [comment]
Thank you

**Comment by ptkdev** [comment]
But you @gajus need re-generate the readme.md with new documentation because on npm appear without require-returns section. 

I don't see require-returns on readme.md

**Comment by gajus** [comment]
Done
